java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -2120,6 +2279,27 @@ public void zpopmax(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    //     TODO: uncomment once client.zadd has binary version
```

Comment:
You can use non-binary `zadd`, but leave a TODO to replace it there with a binary version.
Test should pass since you're not using real binary strings here.

Reply:
Since we are already adding GS version for all functions shortly, I think we should just wait for all implementations to be merged, then go over the TODOs once we are done

Reply:
Please don't forget